src/akamai/js/akamaiHD_flash.js

```diff
@@ -1162,7 +1162,7 @@ var swfobject = function() {
       }
       if (ua.ie && ua.win) { // Internet Explorer + the HTML object element + W3C DOM methods do not combine: fall back to outerHTML
         var att = "";
-        for (var i in attObj) {
+        for (var i=0; i < attObj.length; i++) {
```

Comment:
github isn't updating this when I check in changes, as it used to do in the past.